src/Factories/Logger.php

```diff
@@ -27,14 +27,14 @@ public function __construct(
     }
 
     /**
-     * @param  array<string, mixed>  $config
+     * @param  array<string, mixed>&array{level?: \Psr\Log\LogLevel::*}  $config
```

Comment:
We can provide the default in the `registerLogger` method.

```suggestion
     * @param  array<string, mixed>&array{level: \Psr\Log\LogLevel::*}  $config
```